src/adapter/src/notice.rs

```diff
@@ -179,6 +180,9 @@ impl fmt::Display for AdapterNotice {
             AdapterNotice::BadStartupSetting { name, reason } => {
                 write!(f, "startup setting {name} not set: {reason}")
             }
+            AdapterNotice::RbacDisabled => {
+                write!(f, "RBAC is unimplemented so no role attributes or privileges will be considered when executing statements")
```

Comment:
The role attributes are persisted though, yes? So when we do support them they will suddenly turn on if a user has typed these statements? If that's correct, then I think this message should also talk about that. Maybe:

> RBAC is under development: currently no role attributes or privileges will be considered when executing statements, although these attributes are saved and will be considered in a later release

Reply:
Done